core/src/workflows/hold_link.rs

```diff
@@ -169,12 +165,13 @@ pub mod tests {
             entry: link_entry,
             header,
         };
-
+         println!("before hold link");
```

Comment:
remove printlns?

Reply:
From the broken test still but will remove it